### PR TITLE
add links to the godot-rust project

### PR DESCRIPTION
**Pull request by macalimlim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -589,7 +589,7 @@ See also [Are we (I)DE yet?](https://areweideyet.com/) and [Rust Tools](https://
   * [gitpod.io](https://gitpod.io) — Online IDE with full Rust support based on Rust Language Server
   * [gnome-builder](https://wiki.gnome.org/Apps/Builder) native support for rust and cargo since Version 3.22.2
   * [Kakoune](http://kakoune.org/)
-    * [kak-lsp/kak-lsp](https://github.com/kak-lsp/kak-lsp/) — [LSP](https://microsoft.github.io/language-server-protocol/) client. Implemented in Rust and supports rls out of the box.
+    * [ul/kak-lsp](https://github.com/ul/kak-lsp/) — [LSP](https://microsoft.github.io/language-server-protocol/) client. Implemented in Rust and supports rls out of the box.
```
Comment (macalimlim) [inline README.md:592]: Hmmm I didnt change this... :thinking:

File: README.md [hunk 2]
```diff
@@ -1561,7 +1562,7 @@ A registry allows you to publish your Rust libraries as crate packages, to share
   * [Learning Rust With Entirely Too Many Linked Lists](http://cglab.ca/~abeinges/blah/too-many-lists/book/) — in-depth exploration of Rust's memory management rules, through implementing a few different types of list structures.
   * [Rust by Example](https://doc.rust-lang.org/rust-by-example/)
   * [Rust Cookbook](https://rust-lang-nursery.github.io/rust-cookbook/) — A collection of simple examples that demonstrate good practices to accomplish common programming tasks, using the crates of the Rust ecosystem.
-  * [Rust Online Courses at Classpert](https://classpert.com/it-computer-science/software-development/rust-programming) — A list of Rust online courses (paid) from Classpert Online Course Search
+  * [Rust Online Courses at Classpert](https://classpert.com/rust-programming) — A list of Rust online courses (paid) from Classpert Online Course Search
```
Comment (macalimlim) [inline README.md:1565]: and this... :thinking:

File: README.md [hunk 3]
```diff
@@ -1165,6 +1165,8 @@ See also [Are we game yet?](https://arewegameyet.rs)
   * [oxidator](https://github.com/Ruddle/oxidator) — A real time strategy game/engine written with Rust and WebGPU
   * [Piston](https://www.piston.rs/) — [![Crates.io](https://img.shields.io/crates/v/piston.svg?style=flat-square)](https://crates.io/crates/piston) [![Crates.io](https://img.shields.io/crates/l/piston.svg)](https://github.com/PistonDevelopers/piston/blob/master/LICENSE)
   * [Unrust](https://github.com/unrust/unrust) — unrust — A pure rust based (webgl 2.0 / native) game engine
+* [Godot](https://godotengine.org/)
+  * [godot-rust/godot-rust](https://github.com/godot-rust/godot-rust) [[gdnative](https://crates.io/crates/gdnative)] - Rust bindings to the Godot game engine
```
Comment (palfrey) [inline README.md:1169]: There's a Travis build for godot-rust. Please can you add the build badges to this?

Comment (macalimlim) [reply README.md]: @palfrey done...

File: README.md [hunk 4]
```diff
@@ -1165,6 +1165,8 @@ See also [Are we game yet?](https://arewegameyet.rs)
   * [oxidator](https://github.com/Ruddle/oxidator) — A real time strategy game/engine written with Rust and WebGPU
   * [Piston](https://www.piston.rs/) — [![Crates.io](https://img.shields.io/crates/v/piston.svg?style=flat-square)](https://crates.io/crates/piston) [![Crates.io](https://img.shields.io/crates/l/piston.svg)](https://github.com/PistonDevelopers/piston/blob/master/LICENSE)
   * [Unrust](https://github.com/unrust/unrust) — unrust — A pure rust based (webgl 2.0 / native) game engine
+* [Godot](https://godotengine.org/)
+  * [godot-rust/godot-rust](https://github.com/godot-rust/godot-rust) [[gdnative](https://crates.io/crates/gdnative)] - Rust bindings to the Godot game engine [<https://api.travis-ci.com/godot-rust/godot-rust.svg?branch=master>](https://travis-ci.com/github/godot-rust/godot-rust)
```
Comment (palfrey) [inline README.md:1169]: `<https://api.travis-ci.com/godot-rust/godot-rust.svg?branch=master>` is just a link. It should either be `` or `build badge`

Comment (macalimlim) [reply README.md]: oops :Ddone

Comment (palfrey) [reply README.md]: Closer, but not quite. It's now `<img="` rather than `<img src="`

Comment (macalimlim) [reply README.md]: facepalms myself.... done and I hope for the last time

**Pull request merged**